### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by weronica**
+ Removed "FAQs" page and associated route in `views.py`. 
+ Moved "Resources" drop-down to where "FAQs" used to be.
+ Added `description` field to several of our forms' fields.
+ Added front-end display for required fields (indicated by an *) and a field's description (if applicable.)

### CODE REVIEW

File: app/templates/add_resource.html [hunk 1]
```diff
@@ -31,7 +31,14 @@ <h1 class="page-header">{{ title }}</h1>
...
 		     columns large-centered">
-		      	<label>{{ form.name.label }}
+		    {% if field.flags.required %}
+		      	<label>{{ field.label(class_='required') }}
+		    {% else %}
+				<label>{{ field.label }}
+		    {% endif %}
+		    		<div class="form-field-description">
+		      			{{ field.description }}
+		      		</div>
```
Comment (sandlerben) [inline app/templates/add_resource.html]: The indentation in everything you added in this file looks funky. I'm assuming it's just how github is displaying it.

File: app/templates/guest_add_resource.html [hunk 2]
```diff
@@ -34,66 +34,19 @@ <h1 class="page-header">{{ title }}</h1>
...
 		     columns large-centered">
-		      	<label>{{ form.name.label }}
+		    {% if field.flags.required %}
+		      	<label>{{ field.label(class_='required') }}
+		    {% else %}
+				<label>{{ field.label }}
+		    {% endif %}
+		    		<div class="form-field-description">
+		      			{{ field.description }}
+		      		</div>
```
Comment (sandlerben) [inline app/templates/guest_add_resource.html]: Indentation looks funny here, too.

File: app/templates/add_resource.html (hunk 1 above)
Comment (weronica) [inline app/templates/add_resource.html]: Yes, the indentation looks fine in sublime. I would say that github is just displaying it oddly.

File: app/templates/guest_add_resource.html (hunk 2 above)
Comment (weronica) [inline app/templates/guest_add_resource.html]: Same as above.

**Pull request merged**